### PR TITLE
marcioanatrielo/c01-e03

### PR DESCRIPTION
**Pull request by marcioanatrielo**
# Description

Exercise (e.g. C01-E02): 

Student Name: Marcio Andre

> A brief explanation of what has been done.

Step 1: Create AWS account 
Step 2: Setup Billing 
Step 3: Create Security Group - Devops-Academy 
Step 4: Create SSH key pair Devops-Academy
Step 5: Create SSH key pair Devops-Academy-Jump-Box
Step 6: Lunch Devops00
Step 7: Lunch Devops01 
Step 8: Lunch Devops02 

## Type of change

- [X] Exercise Submission (one exercise per PR)
  - [X] This PR is just for **one** exercise of a class
  - [X] PR name follows the pattern `<github-username>/<exercise-number>`
  - [X] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [X] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-e03/marcio-anatrielo/c01-e03/c01-e03.txt [hunk 1]
```diff
@@ -0,0 +1,64 @@
+##### AWS LAB EXECUTION #####
```
Comment (kikobr82) [inline classes/01class/exercises/c01-e03/marcio-anatrielo/c01-e03/c01-e03.txt]: file is in the wrong folder. it should be your github username.

Comment (kikobr82) [inline classes/01class/exercises/c01-e03/marcio-anatrielo/c01-e03/c01-e03.txt]: file is in the wrong folder. it should be your github username.

Comment (marcioanatrielo) [reply classes/01class/exercises/c01-e03/marcio-anatrielo/c01-e03/c01-e03.txt]: 1) how did you connected to the first public instance; This instance I just select public subnet to be available on public IP by auto. 
2) private IP of the instance on private subnet; I select private network with no public IP after install. After that I add a new interface on my public VM with the same subnet on my private subnet to create a connection between this two VMs. 
3)how did you connected from public to private: Add new interface on public machine to talk at same subnet with my private machine is connected.

Comment (caiocezart) [reply classes/01class/exercises/c01-e03/marcio-anatrielo/c01-e03/c01-e03.txt]: I need you to add the information to the file and update it

Comment (marcioanatrielo) [reply classes/01class/exercises/c01-e03/marcio-anatrielo/c01-e03/c01-e03.txt]: Cool I´m just check if I´m on good way hehehe. If it is correct now I will do this right now!!!

**Pull request merged**